database/db.js

```diff
@@ -13,22 +13,34 @@ const room = mongoose.Schema({
   date: Date,
 });
 
+const roomsSchema = mongoose.Schema({
+  roomName: String,
+  room: [bookingSchema],
+});
+
+const hotelSchema = mongoose.Schema({
+  id: Number,
+  rooms: [roomsSchema],
+});
 
-// in the future I will make a hotel schema with a hotelId containing a child schema called rooms
-// const hotelSchema = mongoose.Schema({
-//   id: Number,
-//   rooms: [room],
-// });
-const Booking = mongoose.model('Booking', room);
-// const Hotel = mongoose.model('Hotel', hotelSchema);
+const Booking = mongoose.model('Booking', bookingSchema);
+const Hotel = mongoose.model('Hotel', hotelSchema);
+const Room = mongoose.model('Room', roomsSchema);
 
 const save = (reservation) => {
-  const booking = new Booking(reservation);
-  booking.save();
+  reservation.save();
```

Comment:
You may want to handle the promise returned upon calling save() function (by using .then or callback)